pkg/registry/registry.go

```diff
@@ -114,7 +114,7 @@ func GetHiddenExposePorts(image string) []model.Port {
 	var err error
 	var username string
 	var token string
-	if okteto.IsOktetoContext() {
+	if okteto.IsOktetoContext() && (strings.HasPrefix(image, okteto.DevRegistry) || strings.HasPrefix(image, okteto.GlobalRegistry)) {
```

Comment:
`image = ExpandOktetoDevRegistry(image)` doesn't change `image` if  `okteto.dev` is not in the string, right?
I think you can remove `image = ExpandOktetoDevRegistry(image)` and leave:
```
		image = ExpandOktetoDevRegistry(image)
		image = ExpandOktetoGlobalRegistry(image)
		username = okteto.Context().UserID
		token = okteto.Context().Token
```